cypress/integration/surveyHomePage.spec.ts

```diff
@@ -0,0 +1,61 @@
+const userProfileDatatestIds = {
+  userName: 'userName',
+  userLine: 'userLine',
+  userAvatar: 'userAvatar',
+  logOutButton: 'loggoutButton',
+  appVersion: 'appVersion',
+  userProfileContainer: 'userProfileContainer',
+};
+const defaultLayoutDataTestIds = {
```

Comment:
Let's add a new line here

```suggestion
};

const defaultLayoutDataTestIds = {
```